app/models/tv_season.rb

```diff
@@ -25,11 +23,12 @@ def self.parse_results(json)
       end
       @overview = result[:overview]
       @still_path = result[:still_path]
+      # TODO: refactor this into a TvEpisode object
```

Comment:
This refactor reaches a little bit outside of the scope of this PR. I'll open another PR to address it.